builder.config.json

```diff
@@ -39,15 +39,15 @@
           "name": "home",
           "path": "\/",
           "methods": ["GET"]
-        }
-      },
-      "trees": {
+        },
         "editTree": {
```

Comment:
@poojahpatil90 I think this will require some changes in voto5 - see the `app/Http/Controllers/FlowInterop/FlowsController.php` index method and `resources/assets/js/floip/builder/views/CustomInteractionDesigner.vue`

Also, how come this is still editTrees rather than editFlows?

Reply:
I think there are also some bits of deep linking config missing from the voto5 flow builder router in `resources/assets/js/floip/builder/routes/index.js`

Reply:
I remember keeping it editFlow and not sure how that changed. I fixed the `editTree` to `editFlow` now.